ecs-configuration/logstash/pipelines/suricata/logstash-601-filter-suricata-smb.conf

```diff
@@ -9,10 +8,13 @@ if [@metadata][stage] == "suricata_json" {
           "[smb][created]" => "[file][created]"
```

Comment:
can you add `smb.accessed` => `file.accessed`?

Reply:
Already in the file.